src/redux/watchList/actionCreator.js

```diff
@@ -13,26 +17,26 @@ export const fetchWatchList = () => ({
   type: FETCH_WATCHLIST_SUCCESS,
 });
 
-export const postWatchList = (payLoad) => ({
+export const postWatchList = (payload) => ({
   type: POST_WATCHLIST_SUCCESS,
-  payLoad,
+  payload,
 });
 
-export const deleteWatchList = (payLoad) => ({
+export const deleteWatchList = (payload) => ({
   type: DELETE_WATCHLIST,
-  payLoad,
+  payload,
 });
 
 export const addStockToWatchlist = (watchlistId, ticker) => ({
   type: ADD_STOCK_TO_WATCHLIST,
-  payLoad: {
+  payload: {
     watchlistId,
     stock: {
       ticker,
       // to suppress the warning, will be removed after watchlist data story
```

Comment:
what was this comment? This is the watchlist story, it shouldn't be here anymore